packages/core/src/components/selectable/selectable.tsx

```diff
@@ -170,6 +173,8 @@ export class Selectable {
               required={this.required}
               checked={this.checked}
               onChange={(e) => this.dsoChange.emit(e)}
+              onBlur={() => (this.keyboardFocus = false)}
+              onKeyUp={() => (this.keyboardFocus = true)}
```

Comment:
Waarom geen `onFocus` ?

Reply:
Dat wordt ook getriggerd door muisklik, maar we willen het alleen met keyboard interactie.

Reply:
🦸